examples/face_recognition_11/face_recognition_11/evaluator.py

```diff
@@ -0,0 +1,234 @@
+
+from kolena.workflow import AxisConfig
+from kolena.workflow import Curve
+from kolena.workflow import CurvePlot
+from kolena.workflow import EvaluationResults
+from kolena.workflow import Histogram
+from kolena.workflow import Plot
+from kolena.workflow import TestCases
+
+
+def compute_threshold(inferences: List[Tuple[GroundTruth, Inference]], fmr: float) -> float:
+    """
+    The threhsold is calculated as Threshold = Quantile_v(1 - FMR) where the quantiles are of the observed
+    IMPOSTER scores, v.
+
+    For more details on how threshold is used see https://pages.nist.gov/frvt/reports/11/frvt_11_report.pdf
+    """
+    similarity_scores = np.array(
+        [inf.similarity if inf.similarity is not None else 0.0 for gt, inf in inferences if not gt.is_same],
+    )
+    threshold = np.quantile(similarity_scores, 1.0 - fmr)
+    return threshold
+
+
+def compute_per_sample(
+    ground_truth: GroundTruth,
+    inference: Inference,
+    threshold: float,
+) -> TestSampleMetrics:
+    is_match, is_false_match, is_false_non_match = False, False, False
+
+    if inference.similarity is None:
+        return TestSampleMetrics(
+            is_match=is_match,
+            is_false_match=is_false_match,
+            is_false_non_match=is_false_non_match,
+            failure_to_enroll=True,
+        )
+
+    if inference.similarity > threshold:  # match
+        if ground_truth.is_same:
+            is_match = True
+        else:
+            is_false_match = True
+    else:  # no match
+        if ground_truth.is_same:
+            is_false_non_match = True
+
+    return TestSampleMetrics(
+        is_match=is_match,
+        is_false_match=is_false_match,
+        is_false_non_match=is_false_non_match,
+        failure_to_enroll=False,
+    )
+
+
+def compute_test_case_metrics(
+    test_samples: List[TestSample],
+    ground_truths: List[GroundTruth],
+    metrics: List[TestSampleMetrics],
+    baseline_fnmr: float,
+) -> TestCaseMetrics:
+    n_genuine_pairs = np.sum([gt.is_same for gt in ground_truths])
+    n_imposter_pairs = np.sum([not gt.is_same for gt in ground_truths])
+
+    n_fm = np.sum([metric.is_false_match for metric in metrics])
+    n_fnm = np.sum([metric.is_false_non_match for metric in metrics])
+    n_fte = np.sum([metric.failure_to_enroll for metric in metrics])
+
+    unique_images = set()
+    for ts in test_samples:
+        unique_images.add(ts.a.locator)
+        unique_images.add(ts.b.locator)
+
+    Δ_fnmr = 0.0
+    if baseline_fnmr != 0.0:
+        Δ_fnmr = (n_fnm / n_genuine_pairs) * 100 - baseline_fnmr
+
+    return TestCaseMetrics(
+        n_images=len(unique_images),
+        n_genuine_pairs=n_genuine_pairs,
+        n_imposter_pairs=n_imposter_pairs,
+        n_fm=n_fm,
+        fmr=(n_fm / n_imposter_pairs) * 100,
+        n_fnm=n_fnm,
+        fnmr=(n_fnm / n_genuine_pairs) * 100,
+        Δ_fnmr=Δ_fnmr,
+        n_fte=n_fte,
+        fter=(n_fte / (n_genuine_pairs + n_imposter_pairs)) * 100,
```

Comment:
Shouldn't `FTER` be `n_fte / n_images` since FTE is at the image level not the pair level?